eth/gen_config.go

```diff
@@ -1,238 +0,0 @@
-// Code generated by github.com/fjl/gencodec. DO NOT EDIT.
```

Comment:
why is this file deleted?

Reply:
Since I changed eth/config.go, this generated file would be different. I think I should generate it locally with [this command](https://github.com/celo-org/celo-blockchain/blob/74d0104bc0d258130677b42e1e6dac0fca88eba0/eth/config.go#L55) and push it maybe.